### PR TITLE
removed unused code 

### PR DESCRIPTION
**Pull request by timbl**
partly to  solve https://github.com/solid/solid-panes/issues/213

### CODE REVIEW

**Comment by timbl** [comment]
(The code with YAHOO was code to use an old Yahoo drag and drop library. That is not used and could be removed.  That removed on  file too.)

**Comment by michielbdejong** [comment]
That commit I just added fixes one of the two problems ^
Travis now gives one  ✔  and one ✘.
I'll look into the other one.

File: package.json [hunk 1]
```diff
@@ -89,7 +89,7 @@
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged",
-      "pre-push": "npm test"
+      "pre-push-disabled": "npm test"
```
Comment (michielbdejong) [inline package.json]: @timbl can you try to run `npm test` now, maybe run `npm install meeting-pane@2.0.0` in your local checkout? If you still see failing tests, can you paste the log?

**Review by michielbdejong — CHANGES_REQUESTED** [review]
Agree with all the removed unused code and added two commits to fix the Travis build which was broken in two ways, but I think the `pre-push` -> `pre-push-disabled` should not be necessary.

File: package.json (hunk 1 above)
Comment (timbl) [reply package.json]: Now passes tests.  Thanks!

**Review by michielbdejong — APPROVED** [review]

**Comment by timea-solid** [comment]
see PR https://github.com/solid/solid-panes/pull/340 as a switch of this work

**Pull request closed without merging**